src/TSID/src/QPFixedBaseTSID.cpp

```diff
@@ -0,0 +1,601 @@
+
+    auto dynamicsParameterHandler = std::make_shared<StdImplementation>();
+
+    dynamicsParameterHandler->setParameter("robot_acceleration_variable_name",
+                                           m_pimpl->robotAccelerationVariable.name);
+    dynamicsParameterHandler->setParameter("joint_torques_variable_name", m_pimpl->jointTorquesVariable.name);
+    dynamicsParameterHandler->setParameter("max_number_of_contacts", 1);
+
+    auto contactGroup = std::make_shared<StdImplementation>();
+    contactGroup->setParameter("variable_name", "base_contact_wrench");
+    contactGroup->setParameter("frame_name", m_pimpl->baseLink);
+    if (!dynamicsParameterHandler->setGroup("CONTACT_0", contactGroup))
+    {
+        log()->error("{} Error setting group for base contact.", logPrefix);
+        return false;
+    }
+
+    if (!m_pimpl->dynamicsTask->initialize(dynamicsParameterHandler))
+    {
+        log()->error("{} Error initializing dynamics task", logPrefix);
+        return false;
+    }
+
+    return true;
+}
+
+bool QPFixedBaseTSID::initialize(std::weak_ptr<ParametersHandler::IParametersHandler> handler)
+{
+    constexpr auto logPrefix = "[QPFixedBaseTSID::initialize]";
+
+    if (!m_pimpl->isKinDynSet)
+    {
+        log()->error("{} The object kinDyn is not set for the base SE3Task.", logPrefix);
+        return false;
+    }
+
+    auto ptr = handler.lock();
+    if (ptr == nullptr)
+    {
+        log()->error("{} The parameter handler is not valid.", logPrefix);
+        return false;
+    }
+
+    if (!ptr->getParameter("verbosity", m_pimpl->isVerbose))
+    {
+        log()->info("{} 'verbosity' not found. The following parameter will be used '{}'.",
+                    logPrefix,
+                    m_pimpl->isVerbose);
+    }
+
+    if (!ptr->getParameter("robot_acceleration_variable_name", m_pimpl->robotAccelerationVariable.name))
+    {
+        log()->error("{} Error while retrieving the robot acceleration variable.", logPrefix);
+        return false;
+    }
+
+    if (!ptr->getParameter("joint_torques_variable_name", m_pimpl->jointTorquesVariable.name))
+    {
+        log()->error("{} Error while retrieving the joint torques variable.", logPrefix);
+        return false;
+    }
+
+    if (!createBaseSE3Task())
+    {
+        log()->error("{} Error creating se3task for the base.", logPrefix);
+        return false;
+    }
+    if (!createDynamicsTask())
+    {
+        log()->error("{} Error creating task for the joint dynamics.", logPrefix);
+        return false;
+    }
+    constexpr std::size_t highPriority = 0;
+    this->addTask(m_pimpl->baseSE3Task, "base_se3_task", highPriority);
+    this->addTask(m_pimpl->dynamicsTask, "dynamics_task", highPriority);
+
+    m_pimpl->isInitialized = true;
+
+    return true;
+}
+
+bool QPFixedBaseTSID::finalize(const System::VariablesHandler& handler)
+{
+    constexpr auto logPrefix = "[QPFixedBaseTSID::finalize]";
+
+    System::VariablesHandler tmp = handler;
```

Comment:
`tmp` here doesn't seem really intutive, since there's also a `tmp in `TaskWithPriority` struct.